src/bug_report.rs

```diff
@@ -0,0 +1,198 @@
+use crate::utils::exec_cmd;
+use std::fs;
+use std::path::PathBuf;
+
+pub fn create() {
+    let os_info = os_info::get();
+
+    let environment = Environment {
+        os_type: os_info.os_type(),
+        os_version: os_info.version().to_owned(),
+        shell_info: get_shell_info(),
+        starship_config: get_starship_config(),
+    };
+
+    let link = make_github_issue_link(crate_version!(), environment);
+
+    if open::that(&link)
```

Comment:
Testing locally, this doesn't seem to work as expected. Despite the browser launching, the "unable to launch your browser" message still appears.

The following works for me though:
```rust
if open::that(&link).is_ok() {
    print!("Take a look at your browser. A GitHub issue has been populated with your configuration")
} else {
    println!(
        "I was unable to launch your browser. You'll have to copy this link instead:\n\n{}",
        link
    );
}
```

Reply:
I just copied this wholesale